src/utils/embed-handler.js

```diff
@@ -20,7 +20,7 @@ function embed({ EMBED_SERVICE } = {}) {
     const { url } = node;
     const props = {
       // prepend the embed service for absolute URLs
-      src: (URI.parse(url).reference === 'absolute' ? EMBED_SERVICE : '') + url,
+      src: (URI.parse(url).reference === 'absolute' ? `${EMBED_SERVICE}/` : '') + url,
```

Comment:
this could be more robust and deal with URLs with or without a trailing slash.....